src/fare-contracts/FareContractInfo.tsx

```diff
@@ -79,6 +83,10 @@ export const FareContractInfoHeader = ({
   const warning = useNonInspectableTokenWarning(preassignedFareProduct?.type);
   const showTwoWayIcon = travelRight.direction === TravelRightDirection.Both;
 
+  const {data: phoneNumber} = useGetPhoneByAccountIdQuery(
+    fareContract?.customerAccountId,
+  );
+
```

Comment:
I think the logic that if a fare contract is sent as a param to `FareContractInfoHeader`, then we know it is a sent contract and should show the phone number is a little too subtle. And it being subtle may cause bugs later when someone needs another value from the fare contract here, and ends up providing it as a param without knowing the side effect it causes.